src/appkit.cs

```diff
@@ -2125,6 +2125,10 @@ public interface NSButton : NSAccessibilityButton {
 		[Mac (10,12)]
 		[Export ("imageHugsTitle")]
 		bool ImageHugsTitle { get; set; }
+
+		[Mac (10,5)]
```

Comment:
Yeah. It was mostly there for "this is new in the header but with an old support number, which was strange enough to note"